plugins/modules/vmware_guest_instant_clone.py

```diff
@@ -84,11 +84,51 @@
       - C(Resources) is the default name of resource pool.
     type: str
     required: False
-
+  guestinfo_vars:
+    description:
+      - Provides GuestOS Customization functionality in instant cloned VM.
+      - A list of key value pairs that will be passed to the destination VM.
+      - These pairs should be used to provide user-defined customization to differentiate the destination VM from the source VM.
+    suboptions:
+      hostname:
+        description:
+          - User specific hostname value which is child of guestinfo.ic option.
+        type: str
+      ipaddress:
+        description:
+          - User specific ipaddress value which is child of guestinfo.ic option.
+        type: str
+      netmask:
+        description:
+          - User specific netmask value which is child of guestinfo.ic option.
+        type: str
+      gateway:
+        description:
+          - User specific gateway value which is child of guestinfo.ic option.
+        type: str
+      dns:
+        description:
+          - User specific dns value which is child of guestinfo.ic option.
+        type: str
+      networktype:
+        description:
+          - User specific networktype value which is child of guestinfo.ic option.
+        type: str
+      uuid:
+        description:
+          - User specific uuid value which is child of guestinfo.ic option.
+        type: str
+      uuidHex:
+        description:
+          - User specific uuidHex value which is child of guestinfo.ic option.
+        type: str
+    default: []
+    type: list
+    elements: dict
 extends_documentation_fragment:
 - community.vmware.vmware.documentation
 
-version_added: '1.9.0'
+version_added: '1.10.0'
```

Comment:
The version_added is the released version for the module.
So, it is fine 1.9.0.

Reply:
Ok cool 👍🏻